src/applications/vaos/tests/new-appointment/components/ReviewPage/index.cc-request.unit.spec.jsx

```diff
@@ -0,0 +1,277 @@
+import React from 'react';
+import moment from 'moment';
+import { expect } from 'chai';
+
+import userEvent from '@testing-library/user-event';
+import { waitFor } from '@testing-library/dom';
+import { Route } from 'react-router-dom';
+
+import {
+  setFetchJSONFailure,
+  mockFetch,
+  resetFetch,
+} from 'platform/testing/unit/helpers';
+import environment from 'platform/utilities/environment';
+
+import { FACILITY_TYPES } from '../../../../utils/constants';
+import {
+  createTestStore,
+  renderWithStoreAndRouter,
+} from '../../../mocks/setup';
+
+import ReviewPage from '../../../../new-appointment/components/ReviewPage';
+import {
+  onCalendarChange,
+  startRequestAppointmentFlow,
+} from '../../../../new-appointment/redux/actions';
+import {
+  mockFacilityFetch,
+  mockMessagesFetch,
+  mockPreferences,
+  mockRequestSubmit,
+} from '../../../mocks/helpers';
+import { getVAFacilityMock } from '../../../mocks/v0';
+
+const initialState = {
+  featureToggles: {
+    vaOnlineSchedulingCancel: true,
+    // eslint-disable-next-line camelcase
+    show_new_schedule_view_appointments_page: true,
+  },
+};
+
+describe('VAOS <ReviewPage> CC request', () => {
+  let store;
+  let start;
+
+  beforeEach(() => {
+    mockFetch();
+    start = moment();
+    store = createTestStore({
+      ...initialState,
+      newAppointment: {
```

Comment:
I ended up mocking out the Redux store here because:

1. I couldn't get our page helpers to work in a beforeEach, and I needed to set up a lot of data three times for both tests. Not sure what's going on there, but might be related to behind the scenes Mocha/RTL stuff
2. For setting form data, most of the Redux actions require the schema for the page along with the data

I did try to set a couple through Redux helpers, but I think for this and the confirmation page, which essentially use all the data in the form, it's hard to do much other than mock out the store.

But we're at least running through all the selectors and testing the submit actions connected together with the page.